modules/home-manager/globals.nix

```
@@ -11,5 +11,9 @@
       default = "teal";
       description = "Global Catppuccin accent";
     };
+
+    enable = (lib.mkEnableOption "Catppuccin globally") // {
+      default = config.catppuccin.enable;
+    };
```

is this not infinite recursion
```suggestion
    enable = lib.mkEnableOption "Catppuccin globally";
```

---

That works fine here.

---

Removing it doesn't have any impact, I'll remove it.